spec/fields/contact_spec.rb

```diff
@@ -10,12 +12,14 @@ def mods_display_contact(mods_record)
       "<mods xmlns=\"http://www.loc.gov/mods/v3\"><note type='contact'>jdoe@example.com</note><note>Note Field</note></mods>"
     ).note
   end
-  it 'should only get contact fields' do
+
+  it 'onlies get contact fields' do
```

Comment:
A little weird auto syntax ¯\\_(ツ)_/¯
```suggestion
  it 'gets contact fields only' do
```

Reply:
"onlies", lol. i love it